src/main/java/org/terasology/thirst/ThirstClientSystem.java

```diff
@@ -22,13 +22,19 @@
 import org.terasology.rendering.nui.NUIManager;
 
 /**
+ * Client system that handles how the Thirst HUD widget component is displayed.
+ *
  * @author Marcin Sciesinski <marcins78@gmail.com>
```

Comment:
You can remove Marcin's author tag, we decided to not use them for the whole project 😉